api/handlers.py

```diff
@@ -1,58 +1,17 @@
 import os
-import uuid
+
 import connexion
-from sqlalchemy import create_engine, Table, Column, String, MetaData, Integer, Boolean, Sequence, BigInteger, ForeignKey, DateTime
-from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy import create_engine
 from sqlalchemy.orm import sessionmaker
-from sqlalchemy.dialects.postgresql import UUID
-from flask_cors import CORS
-
 
-#$Env:DATABASE_URL = $(heroku config:get DATABASE_URL -a found-your-stuff-api);  py.exe handlers.py
+from models import User,Tag
 
 if not os.getenv('DATABASE_URL'):
     os.environ['DATABASE_URL'] = os.popen("heroku config:get DATABASE_URL -a found-your-stuff-api").read().strip() #seems jank
 DATABASE_URL = os.getenv('DATABASE_URL')
 db = create_engine(DATABASE_URL, echo=True)
 Session = sessionmaker(bind=db)
 session = Session()
-Base = declarative_base()
-app = connexion.App(__name__, specification_dir='./')
-cors = CORS(app.app)
-
-
-class User(Base):
-    __tablename__='users'
-    id = Column('id', Integer, primary_key=True)
-    email = Column('email', String, nullable=False)
-    password = Column('password', String, nullable=False)
-    name = Column('name', String)
-    phone_number = Column('phone_number', BigInteger)
-    contact = Column('contact', Boolean, nullable=False)
-    active = Column('active', Boolean, nullable=False)
-
-class Tag(Base):
-    __tablename__="tags"
-    id = Column('id', Integer, primary_key=True)
-    user_id = Column('user_id',Integer, ForeignKey("users.id"), nullable=False)
-    external_id = Column('external_id', UUID(as_uuid=True), nullable=False, default=uuid.uuid4, unique=True)
-    name = Column('name', String)
-    picture = Column('picture', String)
-    active = Column('active', Boolean, nullable=False)
-
-class Message(Base):
-    __tablename__="messages"
-    id = Column('id', BigInteger, primary_key=True)
-    tag_id = Column('tag_id',Integer, ForeignKey("tags.id"), nullable=False)
-    time_stamp = Column('time_stamp', DateTime, nullable=False)
-    body = Column('body', String, nullable=False)
-    picture = Column('picture', String)
-    read = Column('read', Boolean, nullable=False)
-
-def createTables():
-    Base.metadata.create_all(db)
-
-
 
 def getUserByExternalID(external_id):
     currentTag = session.query(Tag).filter(Tag.external_id==external_id).first()
```

Comment:
would be nice to pull all database calls into their own module, and use functions to that module instead

```py
currentTag = database.get_user_by_external_id(external_id)
```

that way when we come to unit testing we can test database functions without testing api functions and vice-versa

Reply:
similarly, both `models.py` and `handlers.py` doing database operations means that we need `sqlalchemy` in both places, this can be consolidated, `handlers` shouldn't care about the database implementation.